Dockerfile

```
@@ -0,0 +1,6 @@
+FROM python:alpine
+RUN python3 -m pip install -U gallery-dl
+RUN python3 -m pip install -U youtube_dl
```

```suggestion
RUN python3 -m pip install -U gallery-dl yt-dlp
```
No point in using 2 commands, and no point in vanilla ytdl.